templates/threatconnect.hbs

```diff
@@ -1,22 +1,59 @@
 {{#if block.detailsLoaded}}
   {{#each results as |orgData orgDataIndex|}}
     {{#if (gt orgDataIndex 0)}}
-      <hr>
+      <hr />
     {{/if}}
 
     <ul class="nav nav-tabs mb-2">
       <li class="nav-item">
-        <a {{action "changeTab" "info" orgDataIndex}} class="nav-link {{if (or (eq orgData.__activeTab "info")(not orgData.__activeTab)) "active"}}" href="#">Info</a>
+        <a
+          {{action "changeTab" "info" orgDataIndex}}
+          class="nav-link {{if (or (eq orgData.__activeTab "info") (not orgData.__activeTab)) "active"}}"
+          href="#"
+        >Info</a>
       </li>
       <li class="nav-item">
-        <a {{action "changeTab" "associations" orgDataIndex}} class="nav-link {{if (eq orgData.__activeTab "associations") "active"}}" href="#">Associations ({{orgData.numAssociations}})</a>
+        <a
+          {{action "changeTab" "associations" orgDataIndex}}
+          class="nav-link {{if (eq orgData.__activeTab "associations") "active"}}"
+          href="#"
+        >Associations ({{orgData.numAssociations}})</a>
       </li>
+      {{#if orgData.dnsInformation.result}}
+        <li class="nav-item">
+          <a
+            {{action "changeTab" "dnsInformation" orgDataIndex}}
+            class="nav-link {{if (eq orgData.__activeTab "dnsInformation") "active"}}"
+            href="#"
+          >DNS Resolutions</a>
+        </li>
+      {{/if}}
     </ul>
 
+    {{#if (eq orgData.__activeTab "dnsInformation")}}
+      <div>
+        <span>{{fa-icon icon="network-wired"}}</span>
+        <span>DNS Resolutions</span>
+      </div>
+      {{#each orgData.dnsInformation.result as |info|}}
+        <div class="dns-information">
+          {{#each info.addresses as |addressField|}}

```

Comment:
Side note, if there is anything you can do UI wise to help show what is different from each result here, I think it could be helpful due to the results all being so similar.  No worries if you can't think of anything as I can't currently either 😅